doc/postgresql.md

```diff
@@ -0,0 +1,34 @@
+# PostgreSQL
+
+[PostgreSQL](https://www.postgresql.org/) is a powerful, open source object-relational database system with over 35 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance.
+
+## Getting Started
+
+```nix
+# In `perSystem.process-compose.<name>`
+{
+  services.postgres."pg1".enable = true;
+}
+```
+
+## Examples
+
+- Run postgres server initialised with a sample database and graphically interact with it using [pgweb](https://github.com/sosedoff/pgweb): <https://github.com/juspay/services-flake/tree/main/example/simple>
+
+## Gotchas
+
+### Unix-domain socket path is too long
+
+We already talk about this in the [data directory guide](https://community.flake.parts/services-flake/datadir#socket-path). In case of postgres, you can set `socketDir` while keeping the `dataDir` unchanged.
```

Comment:
Relative should work I believe?

```suggestion
We already talk about this in the [data directory guide](datadir#socket-path). In case of postgres, you can set `socketDir` while keeping the `dataDir` unchanged.
```

Reply:
And it should be relative path to the `.md` file (not the slug URL).